sql/parser/builtins.go

```diff
@@ -761,3 +773,73 @@ func datumToRawString(datum Datum) (string, error) {
 
 	return "", fmt.Errorf("argument type unsupported: %s", datum.Type())
 }
+
+// Round rounds x to n digits.
+// Directly taken from CPython's `double_round` in
+// cpython/Object/floatobject.c.
```

Comment:
@petermattis Yes, let's stick with the `strconv.FormatFloat` version. I'm closing this PR.

@bdarnell Thanks for the pointer to the python issue.